### ISSUE TITLE
[Feature request] add backup/restore

### ISSUE THREAD

**Issue by barabasz** [pawelmalak/flame#318]
Since docker is recommended installation method and creating (and populating!) beautiful dashboard from scratch is VERY time-consuming there should be backup/restore section in settings. 

This is partially related to #259

**Comment by iAmSaugata** [comment]
Hay, you don't have to create every time from scratch, all you have to do is to keep the database backup manually from the volume/mount point (db.sqlite).
Hopefully backup will introduce later.

**Comment by zilexa** [comment]
It makes little sense to build a backup/restore solution. Please don't waste your time to do that. 

Flame is the portal to your docker containers (and other stuff), hence the docker folder containing the volume mounts should be backed up, as part of a bare essential backup strategy. 

This way you backup each container configuration.

**Comment by MyWay** [comment]
I agree with the backup/restore feature.

**Comment by modrewrite** [comment]
Backup/restore option is indeed needed!

**Comment by skaboy71** [comment]
YES PLEASE!  Needs a backup/restore or an easy API to export JSON of the links like LinkWarden or something.
